lib/ExpressRedisCache.js

```diff
@@ -96,6 +96,10 @@ module.exports = (function () {
         this.emit('disconnected', { host: this.host, port: this.port });
         this.emit('message', 'Disconnected from redis://' + this.client.host + ':' + this.client.port);
       }.bind(this));
+      
+      if ( this.client.connected ) {
+        this.client.emit('connect');
```

Comment:
I think resending `connect` on `client` is not good idea. Because there could be some subscribers that will not expect this.
You should better directly set `self.connected`.